### DISCUSSION TITLE
Add support for referential actions

### DISCUSSION THREAD

**Discussion by JPrisk** [keystonejs/keystone#7727] [Feature requests]
> Referential actions determine what happens to a record when a related record is deleted or updated

> Prisma supports the following referential actions:
> 
> [Cascade](https://www.prisma.io/docs/concepts/components/prisma-schema/relations/referential-actions#cascade)
> [Restrict](https://www.prisma.io/docs/concepts/components/prisma-schema/relations/referential-actions#restrict)
> [NoAction](https://www.prisma.io/docs/concepts/components/prisma-schema/relations/referential-actions#noaction)
> [SetNull](https://www.prisma.io/docs/concepts/components/prisma-schema/relations/referential-actions#setnull)
> [SetDefault](https://www.prisma.io/docs/concepts/components/prisma-schema/relations/referential-actions#setdefault)
[Prisma docs](https://www.prisma.io/docs/concepts/components/prisma-schema/relations/referential-actions)

This feature request would allow users to make cascading deletes as well as a number of other referential actions

- [Initial request](https://keystonejs.slack.com/archives/C01STDMEW3S/p1657097704963999)

**Comment by MohammadKurjieh** [comment]
Any updates on this?
